cypress/features/regression/test/actionPopoverNoIFrame.feature

```diff
@@ -74,7 +75,7 @@ Feature: Action Popover component in noIFrame
       And Action Popover element has golden border on focus
 
   @positive
-  Scenario: verify that the last element sub menu 3 is focused using End key for submenu
+  Scenario: verify that the last sub menu 3 element is focused using End key for submenu
```

Comment:
```suggestion
  Scenario: verify that the last sub menu element is focused using End key for submenu
```

Reply:
I've clarified that the `sub menu 3` is focused